neuralmonkey/vocabulary.py

```diff
@@ -488,6 +500,45 @@ def save_to_file(self, path: str, overwrite: bool = False) -> None:
         with open(path, 'wb') as f_pickle:
             pickle.dump(self, f_pickle)
 
+    def save_wordlist(self, path: str, overwrite: bool = False,
+                      save_frequencies: bool = False):
+        """Save the vocabulary as a wordlist. The file is ordered by the ids of
+        words. This function is used mainly for embedding visualization.
+
+        Arguments:
+            path: The path to save the file to.
+            overwrite: Flag whether to overwrite existing file.
+                       Defaults to False.
+            save_frequencies: store also frequencies?
```

Comment:
Tady bych připsal, že když se to zapne na `True`, tak vyleze tsv s hlavičkou, ať je to vidět v dokumentaci a ne jenom v komentáři v kódu.